### PR TITLE
[zh] `SelfSubjectReview` upgrade version in zh-cn docs

### PR DESCRIPTION
**Pull request by HeavenTonight**
[zh] `SelfSubjectReview` upgrade version in zh-cn docs
Version `authentication.k8s.io/v1alpha1` -> `authentication.k8s.io/v1`

### CODE REVIEW

**Comment by windsonsea** [comment]
Hi @HeavenTonight , this file has some more content in-synced with en upstream. Would you like to include those changes in this PR? You can find those differences from the en upstream by running this command on your local `main` branch:

```
./scripts/lsync.sh content/zh-cn/docs/reference/access-authn-authz/authentication.md
```

**Comment by HeavenTonight** [comment]
> Hi @HeavenTonight , this file has some more content in-synced with en upstream. Would you like to include those changes in this PR? You can find those differences from the en upstream by running this command on your local `main` branch:↳
> 
> ```
> ./scripts/lsync.sh content/zh-cn/docs/reference/access-authn-authz/authentication.md
> ```

Thanks,of course,i will find those differences and sync.

File: content/zh-cn/docs/reference/access-authn-authz/authentication.md [hunk 1]
```diff
@@ -558,17 +558,17 @@ To enable the plugin, configure the following flags on the API server:
...
-| `--oidc-username-claim` | 用作用户名的 JWT 申领（JWT Claim）。默认情况下使用 `sub` 值，即最终用户的一个唯一的标识符。管理员也可以选择其他申领，例如 `email` 或者 `name`，取决于所用的身份服务。不过，除了 `email` 之外的申领都会被添加令牌发放者的 URL 作为前缀，以免与其他插件产生命名冲突。 | sub | 否 |
-| `--oidc-username-prefix` | 要添加到用户名申领之前的前缀，用来避免与现有用户名发生冲突（例如：`system:` 用户）。例如，此标志值为 `oidc:` 时将创建形如 `oidc:jane.doe` 的用户名。如果此标志未设置，且 `--oidc-username-claim` 标志值不是 `email`，则默认前缀为 `<令牌发放者的 URL>#`，其中 `<令牌发放者 URL >` 的值取自 `--oidc-issuer-url` 标志的设定。此标志值为 `-` 时，意味着禁止添加用户名前缀。 | `oidc:` | 否 |
-| `--oidc-groups-claim` | 用作用户组名的 JWT 申领。如果所指定的申领确实存在，则其值必须是一个字符串数组。 | groups | 否 |
-| `--oidc-groups-prefix` | 添加到组申领的前缀，用来避免与现有用户组名（如：`system:` 组）发生冲突。例如，此标志值为 `oidc:` 时，所得到的用户组名形如 `oidc:engineering` 和 `oidc:infra`。 | `oidc:` | 否 |
-| `--oidc-required-claim` | 取值为一个 key=value 偶对，意为 ID 令牌中必须存在的申领。如果设置了此标志，则 ID 令牌会被检查以确定是否包含取值匹配的申领。此标志可多次重复，以指定多个申领。 | `claim=value` | 否 |
-| `--oidc-ca-file` | 指向一个 CA 证书的路径，该 CA 负责对你的身份服务的 Web 证书提供签名。默认值为宿主系统的根 CA。 | `/etc/kubernetes/ssl/kc-ca.pem` | 否 |
-| `--oidc-signing-algs` | 采纳的签名算法。默认为 "RS256"。 | `RS512` | 否 |
+| 参数 | 描述 | 示例                                                                                                                          | 必需？ |
```
Comment (windsonsea) [inline content/zh-cn/docs/reference/access-authn-authz/authentication.md:561]: This change is not necessary, which brings much difficulty for reviewers to identify what is really in-synced.

File: content/zh-cn/docs/reference/access-authn-authz/authentication.md [hunk 2]
```diff
@@ -558,17 +558,17 @@ To enable the plugin, configure the following flags on the API server:
...
+| --------- | ----------- |-----------------------------------------------------------------------------------------------------------------------------| ------- |
+| `--oidc-issuer-url` | 允许 API 服务器发现公开的签名密钥的服务的 URL。只接受模式为 `https://` 的 URL。这通常是提供商的发现URL，更改为空路径。 | 如果颁发者的OIDC发现URL为`https://accounts.provider.example/.well-known/openid-configuration`，则值应为`https://accounts.provider.example` | 是 |
+| `--oidc-client-id` |  所有令牌都应发放给此客户 ID。 | kubernetes                                                                                                                  | 是 |
```
Comment (windsonsea) [inline content/zh-cn/docs/reference/access-authn-authz/authentication.md:564]: Same issue, these spaces are not necessary.

File: content/zh-cn/docs/reference/access-authn-authz/authentication.md [hunk 3]
```diff
@@ -2179,9 +2179,9 @@ You can only make `SelfSubjectReview` requests if:
 -->
 你只能在以下情况下进行 `SelfSubjectReview` 请求：
 
-* 集群启用了 `APISelfSubjectReview` (Beta 版本默认启用)
+* 集群启用了 `APISelfSubjectReview` (Kubernetes版本不需要 {{< skew currentVersion >}}，但较旧的 Kubernetes 版本可能不提供此功能门，或者可能默认将其关闭)
   [特性门控](/zh-cn/docs/reference/command-line-tools-reference/feature-gates/)
-* 集群的 API 服务器已启用 `authentication.k8s.io/v1alpha1` 或者 `authentication.k8s.io/v1beta1`
+* （如果您运行的 Kubernetes 版本低于 v1.28）那么你集群的 API server具有`authentication.k8s.io/v1alpha1` 或者 `authentication.k8s.io/v1beta1`
```
Comment (windsonsea) [inline content/zh-cn/docs/reference/access-authn-authz/authentication.md:2184]:
```suggestion
* （如果你运行的 Kubernetes 版本低于 v1.28）那么你集群的 API 服务器具有 `authentication.k8s.io/v1alpha1` 或者 `authentication.k8s.io/v1beta1`
```
对应注释掉的英文也要修改过来

File: content/zh-cn/docs/reference/access-authn-authz/authentication.md [hunk 4]
```diff
@@ -2179,9 +2179,9 @@ You can only make `SelfSubjectReview` requests if:
...
   {{< glossary_tooltip term_id="api-group" text="API 组" >}}。。
```
Comment (windsonsea) [inline content/zh-cn/docs/reference/access-authn-authz/authentication.md:2185]:
```suggestion
  {{< glossary_tooltip term_id="api-group" text="API 组" >}}。
```

File: content/zh-cn/docs/reference/access-authn-authz/authentication.md [hunk 5]
```diff
@@ -560,7 +560,7 @@ To enable the plugin, configure the following flags on the API server:
 
 | 参数 | 描述 | 示例 | 必需？ |
 | --------- | ----------- | ------- | ------- |
-| `--oidc-issuer-url` | 允许 API 服务器发现公开的签名密钥的服务的 URL。只接受模式为 `https://` 的 URL。此值通常设置为服务的发现 URL，不含路径。例如："https://accounts.google.com" 或 "https://login.salesforce.com"。此 URL 应指向 .well-known/openid-configuration 下一层的路径。 | 如果发现 URL 是 `https://accounts.google.com/.well-known/openid-configuration`，则此值应为 `https://accounts.google.com` | 是 |
+| `--oidc-issuer-url` | 允许 API 服务器发现公开的签名密钥的服务的 URL。只接受模式为 `https://` 的 URL。这通常是提供商的发现URL，更改为空路径。 | 如果颁发者的OIDC发现URL为`https://accounts.provider.example/.well-known/openid-configuration`，则值应为`https://accounts.provider.example` | 是 |
```
Comment (tengqm) [inline content/zh-cn/docs/reference/access-authn-authz/authentication.md:563]:
```suggestion
| `--oidc-issuer-url` | 允许 API 服务器发现公开的签名密钥的服务的 URL。只接受模式为 `https://` 的 URL。这通常是提供商的发现 URL，去掉路径部分。 | 如果颁发者的 OIDC 发现URL为`https://accounts.provider.example/.well-known/openid-configuration`，则值应为 `https://accounts.provider.example` | 是 |
```

File: content/zh-cn/docs/reference/access-authn-authz/authentication.md [hunk 6]
```diff
@@ -746,7 +746,7 @@ Webhook 身份认证是一种用来验证持有者令牌的回调机制。
 * `--authentication-token-webhook-cache-ttl` 用来设定身份认证决定的缓存时间。
   默认时长为 2 分钟。
 * `--authentication-token-webhook-version` 决定是使用 `authentication.k8s.io/v1beta1` 还是
-  `authenticationk8s.io/v1` 版本的 `TokenReview` 对象从 webhook 发送/接收信息。
+  `authenticationk.8s.io/v1` 版本的 `TokenReview` 对象从 webhook 发送/接收信息。
```
Comment (tengqm) [inline content/zh-cn/docs/reference/access-authn-authz/authentication.md:749]:
```suggestion
  `authentication.k8s.io/v1` 版本的 `TokenReview` 对象从 Webhook 发送/接收信息。
```

File: content/zh-cn/docs/reference/access-authn-authz/authentication.md [hunk 7]
```diff
@@ -2179,10 +2179,10 @@ You can only make `SelfSubjectReview` requests if:
 -->
 你只能在以下情况下进行 `SelfSubjectReview` 请求：
 
-* 集群启用了 `APISelfSubjectReview` (Beta 版本默认启用)
+* 集群启用了 `APISelfSubjectReview` (Kubernetes版本不需要 {{< skew currentVersion >}}，但较旧的 Kubernetes 版本可能不提供此功能门，或者可能默认将其关闭)
   [特性门控](/zh-cn/docs/reference/command-line-tools-reference/feature-gates/)
-* 集群的 API 服务器已启用 `authentication.k8s.io/v1alpha1` 或者 `authentication.k8s.io/v1beta1`
-  {{< glossary_tooltip term_id="api-group" text="API 组" >}}。。
+* （如果你运行的 Kubernetes 版本低于 v1.28）那么你集群的 API 服务器具有`authentication.k8s.io/v1alpha1` 或者 `authentication.k8s.io/v1beta1`
```
Comment (tengqm) [inline content/zh-cn/docs/reference/access-authn-authz/authentication.md:2186]:
```suggestion
* （如果你运行的 Kubernetes 版本低于 v1.28）那么你集群的 API 服务器具有
  `authentication.k8s.io/v1alpha1` 或者 `authentication.k8s.io/v1beta1`
```

**Review by tengqm — COMMENTED** [review]
英文的变化要同步到注释中。

File: content/zh-cn/docs/reference/access-authn-authz/authentication.md (hunk 5 above)
Comment (windsonsea) [reply content/zh-cn/docs/reference/access-authn-authz/authentication.md]: Hi @HeavenTonight, if you have some different opinions, please provide feedback. However, it's not a good practice to click `Resolve conversation` when you only accept partial suggestions and keep silent, which may cause much more time to confirm by each reviewer, and might be somewhat impolite.

File: content/zh-cn/docs/reference/access-authn-authz/authentication.md [hunk 8]
```diff
@@ -2172,17 +2172,19 @@ By default, all authenticated users can create `SelfSubjectReview` objects when
...
+* (if you are running a version of Kubernetes older than v1.28) the API server for your
+  cluster has the `authentication.k8s.io/v1alpha1` or `authentication.k8s.io/v1beta1`
   {{< glossary_tooltip term_id="api-group" text="API group" >}}
   enabled.
 -->
 你只能在以下情况下进行 `SelfSubjectReview` 请求：
 
-* 集群启用了 `APISelfSubjectReview` (Beta 版本默认启用)
+* 集群启用了 `APISelfSubjectReview` (Kubernetes版本不需要 {{< skew currentVersion >}}，但较旧的 Kubernetes 版本可能不提供此功能门，或者可能默认将其关闭)
   [特性门控](/zh-cn/docs/reference/command-line-tools-reference/feature-gates/)
```
Comment (windsonsea) [inline content/zh-cn/docs/reference/access-authn-authz/authentication.md:2185]:
```suggestion
* 集群启用了 `APISelfSubjectReview` [特性门控](/zh-cn/docs/reference/command-line-tools-reference/feature-gates/)
  (Kubernetes {{< skew currentVersion >}} 无需启用，但较旧的 Kubernetes 版本可能不提供此特性门控或可能默认为禁用）。
```

File: content/zh-cn/docs/reference/access-authn-authz/authentication.md [hunk 9]
```diff
@@ -2172,17 +2172,19 @@ By default, all authenticated users can create `SelfSubjectReview` objects when
...
-* 集群的 API 服务器已启用 `authentication.k8s.io/v1alpha1` 或者 `authentication.k8s.io/v1beta1`
-  {{< glossary_tooltip term_id="api-group" text="API 组" >}}。。
+* （如果你运行的 Kubernetes 版本低于 v1.28）那么你集群的 API 服务器具有 `authentication.k8s.io/v1alpha1` 或者 `authentication.k8s.io/v1beta1`
```
Comment (windsonsea) [inline content/zh-cn/docs/reference/access-authn-authz/authentication.md:2186]:
```suggestion
* （如果你运行的 Kubernetes 版本低于 v1.28）集群的 API 服务器已启用 `authentication.k8s.io/v1alpha1` 或 `authentication.k8s.io/v1beta1`
```

File: content/zh-cn/docs/reference/access-authn-authz/authentication.md [hunk 10]
```diff
@@ -560,7 +560,7 @@ To enable the plugin, configure the following flags on the API server:
 
 | 参数 | 描述 | 示例 | 必需？ |
 | --------- | ----------- | ------- | ------- |
-| `--oidc-issuer-url` | 允许 API 服务器发现公开的签名密钥的服务的 URL。只接受模式为 `https://` 的 URL。此值通常设置为服务的发现 URL，不含路径。例如："https://accounts.google.com" 或 "https://login.salesforce.com"。此 URL 应指向 .well-known/openid-configuration 下一层的路径。 | 如果发现 URL 是 `https://accounts.google.com/.well-known/openid-configuration`，则此值应为 `https://accounts.google.com` | 是 |
+| `--oidc-issuer-url` | 允许 API 服务器发现公开的签名密钥的服务的 URL。只接受模式为 `https://` 的 URL。这通常是提供商的发现URL，更改为空路径。 | 如果颁发者的OIDC发现URL为 `https://accounts.provider.example/.well-known/openid-configuration` ，则值应为 `https://accounts.provider.example` | 是 |
```
Comment (windsonsea) [inline content/zh-cn/docs/reference/access-authn-authz/authentication.md:563]:
```suggestion
| `--oidc-issuer-url` | 允许 API 服务器发现公开的签名密钥的服务的 URL。只接受模式为 `https://` 的 URL。这通常是提供商的发现 URL，去掉路径部分。 | 如果颁发者的 OIDC 发现 URL为 `https://accounts.provider.example/.well-known/openid-configuration`，则值应为 `https://accounts.provider.example` | 是 |
```

**Review by windsonsea — CHANGES_REQUESTED** [review]
You can directly copy/paste `Suggested change` if you have no more opinions.

File: content/zh-cn/docs/reference/access-authn-authz/authentication.md [hunk 11]
```diff
@@ -746,7 +746,7 @@ Webhook 身份认证是一种用来验证持有者令牌的回调机制。
 * `--authentication-token-webhook-cache-ttl` 用来设定身份认证决定的缓存时间。
   默认时长为 2 分钟。
 * `--authentication-token-webhook-version` 决定是使用 `authentication.k8s.io/v1beta1` 还是
-  `authenticationk8s.io/v1` 版本的 `TokenReview` 对象从 webhook 发送/接收信息。
+  `authenticationk.8s.io/v1` 版本的 `TokenReview` 对象从 Webhook 发送/接收信息。
```
Comment (windsonsea) [inline content/zh-cn/docs/reference/access-authn-authz/authentication.md:749]:
```suggestion
  `authentication.k8s.io/v1` 版本的 `TokenReview` 对象从 Webhook 发送/接收信息。
```

**Pull request closed without merging**